patches/minecraft/net/minecraft/item/ItemStack.java.patch

```diff
@@ -243,3 +243,18 @@
     public SoundEvent func_226629_F_() {
        return this.func_77973_b().func_225520_U__();
     }
+@@ -946,4 +999,14 @@
+    public SoundEvent func_226630_G_() {
+       return this.func_77973_b().func_225519_S__();
+    }
++
++   @Override
++   public CompoundNBT serializeShareTag() {
```

Comment:
This can be moved to the extension interface

Reply:
If I moved this to the extension interface I'd have to make the capability provider serialize method public which it shouldn't be as it's an internal method.